tests/app/forms/test_questionnaire_form.py

```diff
@@ -16,1677 +16,1678 @@
+        schema = load_schema_from_name("test_textfield")
 
-            form = generate_form(
-                schema,
-                question_schema,
-                AnswerStore(),
-                ListStore(),
-                metadata=None,
-                response_metadata={},
-            )
+        question_schema = schema.get_block("name-block").get("question")
 
-            for answer_id in schema.get_answer_ids_for_block("name-block"):
-                self.assertTrue(hasattr(form, answer_id))
+        form = generate_form(
+            schema,
+            question_schema,
+            answer_store,
+            list_store,
+            metadata=None,
+            response_metadata={},
+        )
 
-    def test_form_date_range_populates_data(self):
-        with self.app_request_context():
-            schema = load_schema_from_name("test_date_range")
+        for answer_id in schema.get_answer_ids_for_block("name-block"):
+            assert hasattr(form, answer_id)
 
-            question_schema = schema.get_block("date-block").get("question")
 
-            form_data = MultiDict(
-                {
-                    "date-range-from-answer-day": "01",
-                    "date-range-from-answer-month": "3",
-                    "date-range-from-answer-year": "2016",
-                    "date-range-to-answer-day": "31",
-                    "date-range-to-answer-month": "3",
-                    "date-range-to-answer-year": "2016",
-                }
-            )
+def test_form_date_range_populates_data(app, answer_store, list_store):
+    with app.test_request_context():
+        schema = load_schema_from_name("test_date_range")
 
-            expected_form_data = {
-                "csrf_token": None,
-                "date-range-from-answer": "2016-03-01",
-                "date-range-to-answer": "2016-03-31",
+        question_schema = schema.get_block("date-block").get("question")
+
+        form_data = MultiDict(
+            {
+                "date-range-from-answer-day": "01",
+                "date-range-from-answer-month": "3",
+                "date-range-from-answer-year": "2016",
+                "date-range-to-answer-day": "31",
+                "date-range-to-answer-month": "3",
+                "date-range-to-answer-year": "2016",
             }
-            form = generate_form(
-                schema,
-                question_schema,
-                AnswerStore(),
-                ListStore(),
-                metadata=None,
-                response_metadata={},
-                form_data=form_data,
-            )
+        )
+
+        expected_form_data = {
+            "csrf_token": None,
+            "date-range-from-answer": "2016-03-01",
+            "date-range-to-answer": "2016-03-31",
+        }
+        form = generate_form(
+            schema,
+            question_schema,
+            answer_store,
+            list_store,
+            metadata=None,
```

Comment:
There are few of them , you can simply find and replace
```suggestion
            metadata={},
```